skins/laika/src/components/pages/donation_confirmation/SummaryLinks.vue

```diff
@@ -28,11 +45,14 @@ export default Vue.extend( {
 		donationCanBeCommented(): boolean {
 			return this.confirmationData.donation.paymentType !== 'UEB';
 		},
-		donationCommentUrl(): string {
-			const donation = this.confirmationData.donation;
-			return `/add-comment?donationId=${donation.id}&accessToken=${donation.accessToken}&updateToken=${donation.updateToken}`;
-		},
 	},
+	methods: {
+		openPopUp(): void {
+			if ( !this.$data.disableCommentLink ) {
+				this.$data.addComment = true;
+			}
+		}
+	}
```

Comment:
The state and computed properties are getting confusing to me. What do `addComment` and `disableCommentLink` _mean_? Maybe you can find better names

A test that check if user interactions / events trigger the right state is also missing